docs/source/tools/mythxvsc/index.rst

```diff
@@ -0,0 +1,61 @@
+.. meta::
+   :description: MythXvsc is an extension for running MythX smart contract analysis from your favourite IDE.
+
+.. _tools.mythxvsc:
+
+MythXvsc
+============================
+
+MythXvsc is an extension for running MythX smart contract analysis from your favourite IDE.
+
+The extensions provides:
+
+- Smart contract compilation (via solidity VSCode extension)
+- Log in to MythX platform
+- AST extraction from compiled source
+- Submission of analysis
+- Displaying analysis result in VSCode in a linting fashion
+
+
+
+
+Dependencies
+-----
+
+The MythXvsc extension depends on Juan Blanco’s solidity extension_. Make sure to download this from Visual Studio Code Marketplace before installing MythXvsc.
```

Comment:
I'd make the link text the whole "_Juan Blanco's Solidity extension_" or "the _Solidity extension_ by Juan Blanco"